akka-actor/src/main/scala/akka/actor/ActorRef.scala

```diff
@@ -476,6 +476,33 @@ private[akka] trait MinimalActorRef extends InternalActorRef with LocalRef {
   protected def writeReplace(): AnyRef = SerializedActorRef(this)
 }
 
+/**
+ * An ActorRef that ignores any incoming messages.
+ *
+ * INTERNAL API
+ */
+private[akka] class IgnoreActorRef(systemName: String) extends MinimalActorRef {
+
+  override def provider: ActorRefProvider =
+    throw new UnsupportedOperationException("no provider")
+
+  override def path: ActorPath =
+    RootActorPath(Address("akka", systemName)) / IgnoreActorRef.pathElements
+
+}
+
+/**
+ * INTERNAL API
+ */
+private[akka] object IgnoreActorRef {
+
+  val pathElements: Iterable[String] = immutable.Seq("system", "ignore")
+  val pathString = pathElements.mkString("/", "/", "")
+
+  def isIgnoreRefPath(path: String): Boolean = path.endsWith(pathString)
```

Comment:
This check is fragile. We need to check teh queality for the whoe=le part after the host and port. I will change this.